backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -1319,10 +1319,13 @@ private void setNewSessionExpiration(long timeout) {
     }
 
     private boolean stopImageTransferSession(ImageTransfer entity) {
-        if (entity.getImagedTicketId() == null) {
-            log.warn("Failed to stop image transfer '{}'. Ticket does not exist for image '{}'",
+        if (Guid.isNullOrEmpty(entity.getImagedTicketId())) {
+            log.warn("Image transfer '{}'. Ticket does not exist for image '{}'",
                     getCommandId(), entity.getDiskId());
-            return false;
+
+            // Shouldn't happen, but if the image ticket id is missing in the
+            // database it is very likely that it wasn't created
```

Comment:
Isn't this expected if nbd server could not start?

But returning here is very wrong! the nbd server may be running, and the image may be prepared. We must stop the nbd server and tear down the image.

Reply:
good catch @nirs